plugins/providers/virtualbox/driver/meta.rb

```diff
@@ -32,19 +36,21 @@ def initialize(uuid=nil)
           @logger = Log4r::Logger.new("vagrant::provider::virtualbox::meta")
           @uuid = uuid
 
-          # Read and assign the version of VirtualBox we know which
-          # specific driver to instantiate.
-          begin
-            @version = read_version || ""
-          rescue Vagrant::Errors::CommandUnavailable,
-            Vagrant::Errors::CommandUnavailableWindows
-            # This means that VirtualBox was not found, so we raise this
-            # error here.
-            raise Vagrant::Errors::VirtualBoxNotDetected
+          if !@@version
+            # Read and assign the version of VirtualBox we know which
+            # specific driver to instantiate.
+            begin
+              @@version = read_version
```

Comment:
Ah, VirtualBox doesn't parallelize, but lets just do it to avoid any issues in the future.